pages/opensource.js

```diff
@@ -260,5 +237,10 @@ export async function getStaticProps() {
   const repos = await octokit.paginate('GET /orgs/{org}/repos', {
     org: 'hackclub'
   })
-  return { props: { repos }, revalidate: 30 }
+
+  const transparentAccounts = await fetch(
+    'https://bank.hackclub.com/api/v3/organizations'
+  ).then(res => res.json())
+
+  return { props: { repos, transparentAccounts }, revalidate: 30 }
```

Comment:
hq doesnt generate projects on a half-minute basis so raising this is a good idea (2hours?)
```suggestion
  return { props: { repos, transparentAccounts }, revalidate: 60 * 60 * 2} // 2 hours
```

Reply:
There's no harm in 30 second revalidation, I'd prefer we keep it to 30 seconds